feconf.py

```diff
@@ -51,9 +51,13 @@
 TESTS_DATA_DIR = os.path.join('core', 'tests', 'data')
 SAMPLE_EXPLORATIONS_DIR = os.path.join('data', 'explorations')
 SAMPLE_COLLECTIONS_DIR = os.path.join('data', 'collections')
-INTERACTIONS_DIR = os.path.join('extensions', 'interactions')
-GADGETS_DIR = os.path.join('extensions', 'gadgets')
-RTE_EXTENSIONS_DIR = os.path.join('extensions', 'rich_text_components')
+
+EXTENSIONS_DIR_PREFIX = 'build_staging' if (IS_MINIFIED or not DEV_MODE) else ''
```

Comment:
When backend loads HTML files to be parsed by Jinja and then served they are loaded from core/templates/prod/head, the only reason before this pull request was because they were minified there. This pull request introduces the second reason that is the paths inside the HTML are interpolated with hashes in the prod folder.

Moving that folder is good idea.